### PR TITLE
Introduced new cheat files for Atari Lynx games

### PR DESCRIPTION
**Pull request by superKoder**
- Ninja Gaiden (1990)
- Rampage (1991)

### CODE REVIEW

File: cht/Atari - Lynx/Ninja Gaiden (1990).cht [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cheat0_address = "20244"
+cheat0_address_bit_position = "255"
+cheat0_big_endian = "false"
+cheat0_cheat_type = "1"
+cheat0_code = ""
+cheat0_desc = "my cheat"
```
Comment (RobLoach) [inline cht/Atari - Lynx/Ninja Gaiden (1990).cht:6]: Your cheat?

**Review by RobLoach — CHANGES_REQUESTED** [review]
Thanks for ramping up the Lynx cheats!

File: cht/Atari - Lynx/Ninja Gaiden (1990).cht (hunk 1 above)
Comment (superKoder) [reply cht/Atari - Lynx/Ninja Gaiden (1990).cht:6]: Good catch. Could have sworn I had fixed that. Will do!!

**Comment by superKoder** [comment]
Here you go. Sorry for the delay, got busy at work.

**Pull request merged**